src/__tests__/theme-provider.test.js

```diff
@@ -0,0 +1,108 @@
+/* eslint func-style:0 */
+import React, {Component} from 'react'
+import {render, mount} from 'enzyme'
+import * as jestGlamorReact from 'jest-glamor-react'
+import toJson from 'enzyme-to-json'
+// eslint-disable-next-line import/default
```

Comment:
I think I copied/pasted from the other test file.
I will remove this line from both files as it is not necessary